app/helpers/application_helper.rb

```diff
@@ -15,19 +15,9 @@ def duration_display(minutes)
     hours = minutes / 60
     minutes = minutes - (hours * 60)
 
-    minute_string = minutes > 0 ? "#{minutes}m" : ""
-    hour_string = hours > 0 ? "#{hours}h" : ""
+    minute_string = minutes > 0 ? "#{minutes}" : ""
+    hour_string = hours > 0 ? "#{hours}" : ""
 
-    "#{hour_string} #{minute_string}".strip
-  end
-
-  def long_duration_display(minutes)
-    hours = minutes / 60
-    minutes = minutes - (hours * 60)
-
-    minute_string = minutes > 0 ? "#{minutes} #{'minute'.pluralize(minutes)}" : ""
-    hour_string = hours > 0 ? "#{hours} #{'hour'.pluralize(hours)}" : ""
-
-    "#{hour_string} #{minute_string}".strip
+    "#{hour_string}:#{minute_string}".strip
```

Comment:
Thanks @J3RN. I've added a new PR which formats things with leading zeros. I was able to Postgres (and then the app) running and eyeballed this myself. Now, for instance, a 30 minute duration reads as "00:30" using your specified change. I'd be keen to find a more readable way to write this, but if you're happy, I'm happy.